docs/user/BuildWithDocker.md

```diff
@@ -72,6 +72,41 @@ docker run --rm -it \
            openroad/flow-$OS_NAME-builder
 ```
 
+
+### Docker shell utility
```

Comment:
This is also very similar to the README file in the `flow/util` folder; please refer to the comment below on not duplicating text.

Reply:
I think a sub-entry bellow the build with docker